text/0000-static.md

```diff
@@ -0,0 +1,70 @@
+- Feature Name: static_lifetime_in_statics
+- Start Date: 2016-05-20
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Let's default lifetimes in static and const declarations to `'static`.
+
+# Motivation
+[motivation]: #motivation
+
+Currently, having references in `static` and `const` declarations is cumbersome 
+due to having to explicitly write `&'static ..`. On the other hand anything but 
+static is likely either useless, unsound or both. Also the long lifetime name 
```

Comment:
As I wrote, since `'static` is the only name in scope, it's certainly not "unsound" -- other names are just a guaranteed compilation error. I'd just drop this sentence.